AUTHORS

```diff
@@ -0,0 +1,4 @@
+Mark Zwart <mark.zwart@pobox.com>
+Przemyslaw Socha <sochap@gmail.com>
+Patrick Socha <sochap@gmail.com>
```

Comment:
Hi.
Remove "Patrick Socha", please. It's just me (Przemyslaw Socha) using my middle name ;)
Cheers!